win32/src/winapi/gdi32/object.rs

```diff
@@ -98,6 +99,7 @@ pub fn SelectObject(machine: &mut Machine, hdc: HDC, hGdiObj: HGDIOBJ) -> HGDIOB
             DCTarget::DirectDrawSurface(_) => todo!(),
         },
         Object::Brush(_) => std::mem::replace(&mut dc.brush, hGdiObj),
+        Object::Palette(_) => panic!("SelectObject called with a palette (use SelectPalette)"),
```

Comment:
I guess the author of the exe is unlikely to see this recommendation :)

(The panic is fine, but if a program ends up going down this codepath we'll need to eventually figure out what behavior they expected)